roles/openshift_health_checker/library/ocutil.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+# vim: expandtab:tabstop=4:shiftwidth=4
+"""
+  Interface to OpenShift oc command
+"""
+#
+#   Copyright 2017 Red Hat Inc.
```

Comment:
Most of the source files in this repo don't have this preamble. We do have a LICENSE file in the root of the repo.
I'm not a lawyer, but I'd say unless we have reasons to go this path I'd suggest we keep it simple and remove this.